command/server/config.go

```diff
@@ -60,8 +60,10 @@ func DevConfig(ha, transactional bool) *Config {
 			&Listener{
 				Type: "tcp",
 				Config: map[string]interface{}{
-					"address":     "127.0.0.1:8200",
-					"tls_disable": "1",
+					"address":                      "127.0.0.1:8200",
+					"tls_disable":                  true,
+					"proxy_protocol":               "use_if_authorized",
```

Comment:
See https://github.com/hashicorp/vault/pull/3098/commits/5dea9eb745ea1ee2fb0acef5d75b93b549823d15 for a quick change to address this at the moment.